Rock/Model/AuthService.Partial.cs

```diff
@@ -57,5 +57,19 @@ public IQueryable<Auth> GetAuths( int entityTypeId, int? entityId, string action
             return Get( entityTypeId, entityId )
                 .Where( t => t.Action == action );
         }
+
+        /// <summary>
+        /// Returns an enumerable collection of <see cref="Rock.Model.Auth"/> entities by <see cref="Rock.Model.EntityType"/> and entity Id.
+        /// </summary>
+        /// <param name="groupId">A <see cref="System.Int32"/> represent the groupId of the security Role.</param>
+        /// <returns>
+        /// An enumerable list of <see cref="Rock.Model.Auth" /> entities that secure a specific entity.
+        /// </returns>
+        public IQueryable<Auth> GetAuths( int groupId )
```

Comment:
Can you rename this to `GetByGroup`. Ideally it it would just an overload of the `Get` method with different method signature (parameter list), but with just a single int parameter, need to use a method name that is unique.